dawn/src/dawn/IIR/Extents.cpp

```diff
@@ -24,17 +24,13 @@ namespace iir {
 
 Extent operator+(Extent lhs, Extent const& rhs) { return lhs += rhs; }
 
+Extents::Extents() : Extents(HorizontalExtent{}, Extent{}) {}
 Extents::Extents(HorizontalExtent const& hExtent, Extent const& vExtent)
     : verticalExtent_(vExtent), horizontalExtent_(hExtent) {}
 
 Extents::Extents(const ast::Offsets& offset)
     : verticalExtent_(offset.verticalOffset(), offset.verticalOffset()),
```

Comment:
It looks to me that when you want to initialize a vertical extent you usually want it to correspond to an offset. Therefore it could make sense to add a constructor to `Extent` such as this:
```
Extent(int offset) : minus_(offset), plus_(offset) {}
```
and use it here

Reply:
I see your point. My point against it is that I would not be 100% sure what this constructor does, because it could be either
```
Extent(int offset) : minus_(offset), plus_(offset) {}
```

or

```
Extent(int offset) : minus_(-offset), plus_(offset) {}
```

And both version kind of make sense to me. What do you think?

Reply:
discussed and done.